webots_ros2_driver/src/PythonPlugin.cpp

```diff
@@ -29,17 +29,31 @@ namespace webots_ros2_driver
 
         PyObject *pyWebotsExtraModuleSource = Py_CompileString(
             R"EOT(
-from webots_ros2_driver_webots.controller import Supervisor
+import sys, os
+import webots_ros2_driver_webots
+
+path = os.path.dirname(webots_ros2_driver_webots.__file__)
+
+sys.path.insert(1, path)
+
+from vehicle import Driver
 
 class WebotsNode:
     def __init__(self):
-        self.robot = Supervisor.internalGetInstance()
+        self.robot = Driver.internalGetInstanceDriverIfFeasible()
+
+        # Driver.internalGetInstance == Robot.internalGetInstance
+        if self.robot is None :
+            self.robot = Driver.internalGetInstance()
 )EOT",
             "webots_extra", Py_file_input);
+//self.robot = Car.internalGetInstanceDriver()
```

Comment:
```suggestion
//self.robot = Car.internalGetInstanceDriver()
```
Avoid commenting the code